Kickstarter-iOS/Views/Controllers/PledgeAmountViewController.swift

```diff
@@ -40,16 +36,14 @@ final class PledgeAmountViewController: UIViewController {
       |> ksr_addSubviewToParent()
       |> ksr_constrainViewToEdgesInParent()
 
-    _ = ([self.titleLabel, self.adaptableStackView], self.rootStackView)
+    _ = ([self.titleLabel, self.adaptableStackView, UIView(frame: .zero)], self.rootStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    _ = ([self.stepper, self.spacer, self.amountInputView], self.adaptableStackView)
+    _ = ([self.stepper, self.horizontalSpacer, self.amountInputView], self.adaptableStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
     self.amountInputView.textField.delegate = self
 
-    self.spacer.widthAnchor.constraint(greaterThanOrEqualToConstant: Styles.grid(3)).isActive = true
```

Comment:
We have previously used this width as the minimum space between the stepper and the input field (otherwise when the text field grows it would almost touch the stepper)...now this is done using `spacing` on the stack view which should be much simpler and easier to reason about.